src/app/features/creature/creature-template-resistance/creature-template-resistance.component.ts

```diff
@@ -0,0 +1,30 @@
+import { Component } from '@angular/core';
+
+import { SingleRowEditorComponent } from '@keira-abstract/components/editors/single-row-editor.component';
+import { CreatureHandlerService } from '../creature-handler.service';
+import { CreatureTemplateResistance } from '@keira-types/creature-template-resistance.type';
+import { CreatureTemplateResistanceService } from './creature-template-resistance.service';
+import { CREATURE_TEMPLATE_RESISTANCE_TABLE } from '@keira-types/creature-template-resistance.type';
+import { SPELLSCHOOLRES } from '@keira-constants/options/spell-school-res';
+
+@Component({
+  selector: 'keira-creature-template-resistance',
+  templateUrl: './creature-template-resistance.component.html',
+  styleUrls: ['./creature-template-resistance.component.scss']
```

Comment:
I know we did it too in existing files, but it's better to avoid including empty files like this. So you can just remove this line and delete the scss file.

Reply:
This made the build fail locally.

Reply:
you need to remove the line as well 

```
styleUrls: ['./creature-template-resistance.component.scss']
```

Reply:
yea I did, but reverted it now